README.md

```diff
@@ -25,17 +25,17 @@ For details, see the User's Manuals:
       [the compiler library](https://hackage.haskell.org/package/bond) that
       powers `gbc`.
 
-For a discussion how Bond compares to similar frameworks see [Why Bond](https://microsoft.github.io/bond/why_bond.html).
+For a discussion about how Bond compares to similar frameworks see [Why Bond](https://microsoft.github.io/bond/why_bond.html).
 
 ## Dependencies
 
 Bond C++ library requires some C++11 features (currently limited to those
-supported bv Visual C++ 2013); a C++11 compiler is required. Additionally,
+supported by Visual C++ 2013); a C++11 compiler is required. Additionally,
 to build Bond you will need CMake (3.1+),
 [Haskell Stack](https://docs.haskellstack.org/en/stable/README/#how-to-install)
 (1.5.1+) and Boost (1.61+).
 
-Additionally, Bond requires RapidJSON and optionally requires gRPC. The Bond repository primarily uses Git submodules for these two dependencies. It should be cloned with the `--recursive` flag:
+Additionally, Bond requires RapidJSON and optionally requires gRPC. The Bond repository primarily uses Git submodules for these two dependencies. It should be closed with the `--recursive` flag:
```

Comment:
"cloned" was correct here: it's the Git command that needs to be used. Can you revert this change?